lineman/app/components/memberships_page/memberships_page_controller.coffee

```diff
@@ -1,15 +1,27 @@
-angular.module('loomioApp').controller 'MembershipsPageController', ($scope, group, Records, CurrentUser) ->
-  $scope.group = group
-  Records.memberships.fetchByGroup group
+angular.module('loomioApp').controller 'MembershipsPageController', ($routeParams, Records, LoadingService, CurrentUser) ->
+  @loading = true
+  @loadedCount = 0
+  @membershipsPerPage = 25
 
-  $scope.userIsAdmin = ->
-    CurrentUser.isAdminOf($scope.group)
+  Records.groups.findOrFetchByKey($routeParams.key).then (group) =>
+    @group = group
+    @loadMore()
 
-  $scope.toggleMembershipAdmin = (membership) ->
+  @userIsAdmin = =>
+    CurrentUser.isAdminOf(@group)
+
+  @loadMore = =>
```

Comment:
I don't think (bordering on sure that) you don't need fat arrow on the method definition line. Is there a reason or just playing safe?